packages/react-scripts/config/paths.js

```diff
@@ -80,13 +80,13 @@ module.exports = {
 if (__dirname.indexOf(path.join('packages', 'react-scripts', 'config')) !== -1) {
   module.exports = {
     appBuild: resolveOwn('../../../build'),
-    appPublic: resolveOwn('../template/public'),
-    appHtml: resolveOwn('../template/public/index.html'),
-    appIndexJs: resolveOwn('../template/src/index.js'),
+    appPublic: resolveOwn('../templates/app/public'),
```

Comment:
I want to keep this path as `/template`. Otherwise we'll break User Guide URL once again.
It's fine to add another top-level folder, e.g. `/fixtures`.